### PR TITLE
tools: add fpgainfo fme verbose support

### PR DESCRIPTION
**Pull request by anandaravuri**

### CODE REVIEW

File: tools/fpgainfo/fmeinfo.c [hunk 1]
```diff
@@ -43,6 +58,89 @@ void fme_help(void)
...
 }
 
+static void print_fme_verbose_info(fpga_token token)
+{
+	fpga_result res = FPGA_OK;
+	fpga_object fpga_object;
+	uint64_t bitstream_id;
+	uint32_t major = 0;
+	uint32_t val = 0;
+	printf("-----FME VERBOSE-----\n");
```
Comment (figozhang) [inline tools/fpgainfo/fmeinfo.c:68]: we can remote this printf?

Comment (anandaravuri) [reply tools/fpgainfo/fmeinfo.c:68]: fixed , removed printf

File: tools/fpgainfo/fmeinfo.c [hunk 2]
```diff
@@ -111,6 +210,10 @@ fpga_result fme_command(fpga_token *tokens, int num_tokens, int argc,
 			fme_help();
 			return FPGA_INVALID_PARAM;
 
+		case 'v': /* verbose - UNDOCUMENTED */
```
[the remark this answers is not in the document]
Comment (anandaravuri) [reply tools/fpgainfo/fmeinfo.c:213]: remove comment undocumented

File: tools/fpgainfo/fmeinfo.c [hunk 3]
```diff
@@ -43,6 +58,89 @@ void fme_help(void)
...
+	if (val < sizeof(phase) / sizeof(char *))
+		printf("%s\n", phase[val]);
+	else
+		printf("unknown\n");
+
+	val = FPGA_BSID_REVISION(bitstream_id);
+	printf("%-32s : %03x\n", "Revision", val);
+
+	val = FPGA_BSID_INTERFACE(bitstream_id);
+	if (major == 0) {	// Vista Creek
```
Comment (tswhison) [inline tools/fpgainfo/fmeinfo.c:105]: s,Vista Creek,N3000,

File: tools/fpgainfo/fmeinfo.c [hunk 4]
```diff
@@ -82,10 +180,11 @@ fpga_result fme_command(fpga_token *tokens, int num_tokens, int argc,
 	(void)argv;
 
 	fpga_result res = FPGA_OK;
-
+	int verbose_opt = 0;
 	optind = 0;
 	struct option longopts[] = {
 		{"help", no_argument, NULL, 'h'},
+		{"verbose", no_argument, NULL, 'v'},
```
Comment (tswhison) [inline tools/fpgainfo/fmeinfo.c:187]: Can we use V here instead of v (version)?

File: tools/fpgainfo/fmeinfo.c (hunk 3 above)
Comment (anandaravuri) [reply tools/fpgainfo/fmeinfo.c]: Changed comments to PAC n3000

File: tools/fpgainfo/fmeinfo.c (hunk 4 above)
Comment (anandaravuri) [reply tools/fpgainfo/fmeinfo.c]: verbose changes to V

**Pull request merged**